File: BankServicePlatform/model/customer.py
__author__ = 'Johnny'
# -*- coding: utf-8 -*-
from ..core import db

class Customer(db.model):
    __tablename__="customer"

    id = db.Column(db.Integer(), primary_key=True)

    password=db.Column(db.String(32))
    real_name=db.Column(db.String(8))#姓名
    identification_number=db.Column(db.String(20))#身份证
    phone=db.Column(db.String(20))#手机号码
    channel=db.Column(db.Integer())#渠道
    create_date=db.Column(db.DateTime())#创建时间

File: BankServicePlatform/frontend/static/js/customer.js
/**
 * Created by Johnny on 2017/8/7 0007.
 */

var Customer = Backbone.Model.extend({
    urlRoot: '/api/customer'
});

var Customers = Backbone.Collection.extend({
    url: '/api/customer',
    model: Customer,
    parse: function(response) {
        return response.data;
    }
});

var CustomerRouter = Backbone.Router.extend({
    routes: {
        "customer/:customer_id": "showCustomer"
    },

    showCustomer: function(customer_id) {

    }

});


var customers = new Customers();

customers.fetch({
    success: function(collection, response, options) {

    },
    error: function(collection, response, options) {

    }
});

var newCustomer = customers.create({
    real_name: $('#real_name'),
    phone: $('#phone'),
    identification_number: $('#identification_number'),
    password: $('#password'),
    channel: '1'
});